spk/shairport-sync/src/shairport-sync.sc

```diff
@@ -8,4 +8,4 @@ dst.ports="8303/tcp"
 title="Shairport Sync UDP"
 desc="Shairport Sync UDP"
 port_forward="yes"
-dst.ports="8303:8313/udp"
+dst.ports="8304:8313/udp"
```

Comment:
No idea if this is temporary or definitive - but in any case, do not forget to update port documented at: https://github.com/SynoCommunity/spksrc/wiki/SynoCommunity-Used-Ports